cypress/fixtures/test-import.yaml

```diff
@@ -0,0 +1,161 @@
+apiVersion: provisioning.cattle.io/v1
+kind: Cluster
+metadata:
+  annotations:
+    field.cattle.io/creatorId: user-4rzxn
+  creationTimestamp: "2022-07-04T13:59:52Z"
+  finalizers:
+  - wrangler.cattle.io/provisioning-cluster-remove
+  - wrangler.cattle.io/rke-cluster-remove
+  generation: 2
+  managedFields:
```

Comment:
the `managedFields` can be removed as it is not required when a resource is created

Reply:
I'm actually removing this file, as it was not meant to be used anymore 😬